CMake/fletch-tarballs.cmake

```diff
@@ -311,15 +311,15 @@ if(NOT WIN32)
 endif()
 
 # Caffe
-if(NOT WIN32)
-  set(Caffe_version "rc4")
-  set(Caffe_url "https://github.com/BVLC/caffe/archive/${Caffe_version}.tar.gz")
-  set(Caffe_md5 "d86eeb38b1400097d32ffcabdec75b55")
+if(WIN32)
+  set(Caffe_version "527f97c0692f116ada7cb97eed8172ef7da05416")
+  set(Caffe_url "https://gitlab.kitware.com/kwiver/caffe/repository/archive.zip?ref=${Caffe_version}")
+  set(Caffe_md5 "4ec71f28a797eac7fe3ddcb0fbfab60e")
   list(APPEND fletch_external_sources Caffe)
 else()
-  set(Caffe_version "527f97c0692f116ada7cb97eed8172ef7da05416")
-  set(Caffe_url "https://github.com/BVLC/caffe/archive/${Caffe_version}.zip")
-  set(Caffe_md5 "751e3cd16050e7ecd3981457e41b29cd")
+  set(Caffe_version "7f5cea3b2986a7d2c913b716eb524c27b6b2ba7b")
+  set(Caffe_url "https://gitlab.kitware.com/kwiver/caffe/repository/archive.zip?ref=${Caffe_version}")
```

Comment:
We can follow it on our local and have more control in case we need to point caffe to a merger of two branches, as I need on VIAME